demos/security_barrier_camera_demo/main.cpp

```diff
@@ -810,30 +844,32 @@ int main(int argc, char* argv[]) {
         }
 
         // Running
-        context.t0 = std::chrono::steady_clock::now();
         worker->runThreads();
+        context.t0 = std::chrono::steady_clock::now();
         worker->threadFunc();
         worker->join();
         const auto t1 = std::chrono::steady_clock::now();
 
+        std::map<std::string, std::string> mapDevices = getMapFullDevicesNames(ie, {FLAGS_d, FLAGS_d_va, FLAGS_d_lpr});
         for (auto& net : std::array<std::pair<std::vector<InferRequest>, std::string>, 3>{
             std::make_pair(context.detectorsInfers.getActualInferRequests(), FLAGS_d),
                 std::make_pair(context.attributesInfers.getActualInferRequests(), FLAGS_d_va),
                 std::make_pair(context.platesInfers.getActualInferRequests(), FLAGS_d_lpr)}) {
             for (InferRequest& ir : net.first) {
                 ir.Wait(IInferRequest::WaitMode::RESULT_READY);
                 if (FLAGS_pc) {  // Show performace results
-                    printPerformanceCounts(ir, std::cout, getFullDeviceName(mapDevices, net.second));
+                    printPerformanceCounts(ir, std::cout, std::string::npos == net.second.find("MULTI") ? getFullDeviceName(mapDevices, net.second)
+                                                                                                        : net.second);
                 }
             }
         }
 
         uint64_t frameCounter = context.frameCounter;
         if (0 != frameCounter) {
-            const ms meanOverallTimePerAllInputs = std::chrono::duration_cast<ms>((t1 - context.t0)
+            const Ms meanOverallTimePerAllInputs = std::chrono::duration_cast<Ms>((t1 - context.t0)
                 * context.readersContext.inputChannels.size()) / frameCounter;
-            std::cout << "Mean overall time per all inputs: " << std::fixed << std::setprecision(2) << meanOverallTimePerAllInputs.count()
-                      << "ms / " << std::chrono::seconds(1) / meanOverallTimePerAllInputs << "FPS for " << frameCounter << " frames\n";
+            std::cout << std::fixed << std::setprecision(1) << std::chrono::seconds{1} / meanOverallTimePerAllInputs
```

Comment:
Something is fishy here. You're dividing time by time, which results in a dimensionless quantity. Yet you've labeled it "FPS" AKA Hz.

Reply:
rewrote